mycroft/audio/cache_handler.py

```diff
@@ -0,0 +1,130 @@
+# Cache handler - reads all the .dialog files (The default
+# mycroft responses) and does a tts inference.
+# It then saves the .wav files to mark1 device
+
+import base64
+import glob
+import os
+import re
+import shutil
+import hashlib
+import json
+from urllib import parse
+from requests_futures.sessions import FuturesSession
+from mycroft.util.log import LOG
+
+
+REGEX_SPL_CHARS = re.compile('[@#$%^*()<>/\|}{~:]')
+MIMIC2_URL = 'https://mimic-api.mycroft.ai/synthesize?text='
+
+# Check for more default dialogs
+res_path = '/opt/venvs/mycroft-core/lib/python3.4/' \
+           'site-packages/mycroft/res/text/en-us/'
+wifi_setup_path = '/usr/local/mycroft/mycroft-wifi-setup/dialog/en-us/'
+cache_dialog_path = [res_path, wifi_setup_path]
+# Path where cache is stored and not cleared on reboot/TTS change
+cache_audio_dir = '/opt/mycroft/preloaded_cache'
```

Comment:
This directory should be configurable. In a packaging situation we'd probably like to place it somewhere in /usr/share while on a desktop / github-install /opt/mycroft would be best.